doc/parameters/param_list.md

```diff
@@ -120,7 +120,8 @@ When `plugins` parameter is not overridden, the following default plugins are lo
 | `<data source>`.marking | true | Whether source should mark in costmap |
 | `<data source>`.clearing | false | Whether source should raytrace clear in costmap |
 | `<data source>`.obstacle_range | 2.5 | Maximum range to mark obstacles in costmap |
-| `<data source>`.raytrace_range | 3.0 | Maximum range to raytrace clear obstacles from costmap |
+| `<data source>`.raytrace_max_range | 3.0 | Maximum range to raytrace clear obstacles from costmap |
```

Comment:
Please also update the names of these in the `nav2_bringup/config/*` files so that those have these new parameters reflected. 

We also need a PR in https://github.com/ros-planning/navigation.ros.org to reflect the parameter change in the costmap configuration pages and migration guide that this has changed.

Reply:
I found the pages for you for reference
- https://github.com/ros-planning/navigation.ros.org/blob/master/configuration/packages/costmap-plugins/obstacle.rst
- https://github.com/ros-planning/navigation.ros.org/blob/master/configuration/packages/costmap-plugins/voxel.rst
- https://github.com/ros-planning/navigation.ros.org/blob/master/migration/Foxy.rst

Reply:
Add a new section to the migration page just explaining the parameter name change and that there's a minimum one too - link to this PR and the ROS answers ticket

Reply:
Thanks for the info! I am not able to find any folder named config in the nav2_bringup package. I think I have updated all references to the old parameters (raytrace_range)

Reply:
Sorry, params, https://github.com/ros-planning/navigation2/tree/main/nav2_bringup/bringup/params

I use those interchangably. I should probably figure out which is the correct one...

Problems for another day :smile: